sentry/src/main/java/io/sentry/SpanId.java

```diff
@@ -1,52 +1,53 @@
 package io.sentry;
 
-import io.sentry.util.Objects;
+import io.sentry.util.LazyEvaluator;
 import io.sentry.util.StringUtils;
 import java.io.IOException;
 import java.util.UUID;
 import org.jetbrains.annotations.NotNull;
 
 public final class SpanId implements JsonSerializable {
-  public static final SpanId EMPTY_ID = new SpanId(new UUID(0, 0));
+  public static final SpanId EMPTY_ID = new SpanId("00000000-0000-0000-0000-000000000000");
 
-  private final @NotNull String value;
+  private final @NotNull LazyEvaluator<String> lazyValue;
 
   public SpanId(final @NotNull String value) {
-    this.value = Objects.requireNonNull(value, "value is required");
+    this.lazyValue = new LazyEvaluator<>(() -> value);
```

Comment:
Should we keep the check?

```suggestion
		Objects.requireNonNull(value, "value is required")
    this.lazyValue = new LazyEvaluator<>(() -> value);
```